### DISCUSSION TITLE
Combine updates of multiple managers?

### DISCUSSION THREAD

**Discussion by lauraseidler** [renovatebot/renovate#8447] [Request Help] [answered]
We'd like to combine updates of multiple managers (in this case, `dockerfile` and `npm`), as the dependencies are related.

Example: There's a docker image `cypress/included` which pre-installs the Cypress binaries, and an npm package `cypress` which installs the npm package. The version of these two dependencies should match and be updated in the same PR.

However, I can't seem to figure out the grouping configuration. I've tried:
```
{
  "packageRules": [
    {
      "groupName": "Cypress",
      "managers": [
        "dockerfile",
        "npm"
      ],
      "packagePatterns": [
        "^cypress/included$",
        "^cypress$"
      ]
    }
  ]
}
  ```
  without success. I also tried without the `managers`, with `datasources` instead, and with various package patterns, but still multiple branches/PRs are created.
  
We have a test repo here: https://github.com/eyalzek/renovate-test (with some other test configuration that are not related to this problem).

If anyone managed to make this work, I would be very grateful for some pointers, thanks!

**Comment by rarkins** [comment, marked as answer]
Try this:

```
{
  "packageRules": [
    {
      "groupName": "Cypress",
      "managers": [
        "dockerfile",
        "npm"
      ],
      "additionalBranchPrefix": '',
      "packagePatterns": [
        "^cypress/included$",
        "^cypress$"
      ]
    }
  ]
}
```

`additionalBranchPrefix` is what's adding the `docker-` part after `renovate/` in the Docker branch names, so this should hopefully reset it to empty.

**Reply by lauraseidler** [reply]
Indeed that works - thanks for the _super_ fast reply!
